### PR TITLE
Add some documentation

### PR DESCRIPTION
**Pull request by HoverEpic**
Start new documentation about how proxy works and packets translating.

### CODE REVIEW

File: wiki/Packets.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+| Player Action | 0x24 | 36 |  |  |  |  |  |
+| Entity Fall | 0x25 | 37 |  |  |  |  |  |
+| Hurt Armor | 0x26 | 38 |  |  |  |  |  |
+| Set Entity Data | 0x27 | 39 |  | S => C | Entity Properties | 0x4E |  |
+| Set Entity Motion | 0x28 | 40 |  |  |  |  |  |
+| Set Entity Link | 0x29 | 41 |  | S => C | Set Passengers | 0x43 |  |
+| Set Health | 0x2a | 42 |  | S => C | Update Health | 0x41 |  |
+| Set Spawn Position | 0x2b | 43 |  | S => C | Spawn Position | 0x46 |  |
+| Animate | 0x2c | 44 |  | S => C | Animation | 0x06 |  |
+| Respawn | 0x2d | 45 |  | S => C | RespawnPacket | 0x35 |  |
```
Comment (lukeeey) [inline wiki/Packets.md]: Remove the "Packet" text

File: wiki/Packets.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+| Crafting Data | 0x34 | 52 |  |  |  |  |  |
+| Crafting Event | 0x35 | 53 |  |  |  |  |  |
+| Gui Data Pick Item | 0x36 | 54 |  |  |  |  |  |
+| Adventure Settings | 0x37 | 55 |  | S => C | Change Game State | 0x1E |  |
+| Block Entity Data | 0x38 | 56 |  | S => C | Update Block Entity | 0x09 |  |
+| Player Input | 0x39 | 57 |  | C => S | Steer Vehicle | 0x16 |  |
+| Full Ch Data | 0x3a | 58 |  | S => C | Chunk Data | 0x20 |  |
+| Set Commands Enabled | 0x3b | 59 |  |  |  |  |  |
+| Set Difficulty | 0x3c | 60 |  | S => C | Server Difficulty | 0x0D |  |
+| Change Dimension | 0x3d | 61 |  | S => C | RespawnPacket | 0x35 |  |
```
Comment (lukeeey) [inline wiki/Packets.md]: same as above

File: wiki/Welcome.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+## Welcome do DragonProxy WIKI
```
Comment (lukeeey) [inline wiki/Welcome.md]: to*

Comment (lukeeey) [reply wiki/Welcome.md]: Also, the wiki text doesnt need to be in full caps

File: wiki/Welcome.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+### 1. Presentation
+DragonProxy aim to be the first Minecraft PE <=> PC bridge, based on protocole translation.<br />
```
Comment (lukeeey) [inline wiki/Welcome.md]: aims* 
protocol*

to best the best* (you cant aim to be the first. you either are, or you arent. and we are.)

Minecraft: Bedrock Edition <=> Minecraft: Java Edition

File: wiki/Welcome.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+With this proxy, PE players (W10, Android / IOS) can join PC servers.<br />
+This is probably the future of Minecraft servers, as Microsoft does not provide featured PE server publicly !
```
Comment (lukeeey) [inline wiki/Welcome.md]: Lol you realise featured servers just use regular third party server softwares like PocketMine-MP

File: wiki/Welcome.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+### 2. How it works
+Here is a small description of the protocole translator :<br />
```
Comment (lukeeey) [inline wiki/Welcome.md]: protocol*

File: wiki/Welcome.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+- 1. [JRakNet](https://github.com/JRakNet/JRakNet)<br />
+This is a java based Raknet protocole implementation.
```
Comment (lukeeey) [inline wiki/Welcome.md]: protocol*

**Review by lukeeey — CHANGES_REQUESTED** [review]

File: wiki/Welcome.md (hunk 5 above)
Comment (HoverEpic) [reply wiki/Welcome.md]: I was talking about server core, PocketMine-MP is "fan-made" and does not provide a good server as the java edition.

File: wiki/Welcome.md (hunk 7 above)
Comment (HoverEpic) [reply wiki/Welcome.md]: That damned protocol ^^

File: wiki/Welcome.md (hunk 4 above)
Comment (HoverEpic) [reply wiki/Welcome.md]: Thanks, my english is so bad sometimes ><

**Comment by lukeeey** [comment]
This is probably not gonna be the future of servers since its a proxy but meh

**Comment by DefinitlyEvil** [comment]
DragonProxy not for servers, it's more kinda like for players

**Comment by HoverEpic** [comment]
It can be for both, the MCProtocolLib and JRaknet libraries have realy high performance.

**Comment by DefinitlyEvil** [comment]
well if MovePlayerPacket won't work, DragonProxy is just a s**t

**Comment by HoverEpic** [comment]
I'm on it, I have made dumps if you want.

**Comment by DefinitlyEvil** [comment]
I make dumps too bro :(

**Pull request merged**